sdk/nodejs/runtime/closure/serializeClosure.ts

```diff
@@ -94,12 +95,13 @@ export async function serializeFunction(
 }
 
 /**
- * @deprecated This function has been replaced by `serializeFunction`, which accepts additional parameters and returns
- * more details about the serialized function.  This form will be removed in a future release of this package.
+ * @deprecated Please use 'serializeFunction' instead."
  */
 export async function serializeFunctionAsync(
         func: Function,
         serialize?: (o: any) => boolean): Promise<string> {
+    log.warn("'function serializeFunctionAsync' is deprecated.  Please use 'serializeFunction' instead.");
```

Comment:
same pattern.  in teh log message, saying the symbol that is deprecated.  then say what to do.  The latter sentence should match what is in `@deprecated`.   We can keep things simple.